### DISCUSSION TITLE
balanceOf(acc, nftId) Returns 0, and Suggestions on How to Write Frontend Interactions with a Reach SmartContract.

### DISCUSSION THREAD

**Discussion by BdVade** [reach-sh/reach-lang#1031] [Q&A]
I transferred a testnet NFT to a wallet and I'm trying to check the account's balance for that particular ASA. I keep getting 0 on the front end. I'm also having issues figuring out how to interact with the smart contract from the front end. I'd appreciate any help. The code can be found here : [repo](https://github.com/BdVade/TimedAuction/tree/main/src)

**Comment by jeapostrophe** [comment]
Can you point more exactly at the code in the repo that isn't working? (The line number, for instance.) My hypothesis with your issue is that there's a problem with the networks or ASAs not being the same in the testing vs non-testing environment.
